package.json

```diff
@@ -4,6 +4,44 @@
   "description": "Core Components Library Developed & Maintained By The Articles and Reach & Languages Team",
   "main": "index.js",
   "private": true,
+  "workspaces": [
+    "@bbc/psammead-amp-geo", 
```

Comment:
Does this work? I thought you needed to specify the directory of a workspace? i.e. `/packages/components/*`

Reply:
Following these instructions https://classic.yarnpkg.com/en/docs/workspaces/ it seems like the package name just needs to match (rather than location) 🤷‍♀️

Reply:
The yarn.lock file looks correct too 😄